src/components/com_tjcertificate/administrator/views/template/tmpl/edit.php

```diff
@@ -31,11 +31,18 @@
 $app = Factory::getApplication();
 $input = $app->input;
 
-$client = $input->getCmd('client', '');
+$client    = $input->getCmd('client', '');
+$extension = $input->getCmd('extension', '');
 
-if (!empty($client))
+$clientUrlAppend = '';
```

Comment:
@praneettekdi Avoid data manipulation in the view files

Reply:
its fine if we manipulating URL for form submission